### PR TITLE
Frontend pusher- Now works on both frontend and backend

### PR DESCRIPTION
**Pull request by Jaffenheimer**
Need to create a file called "secrets.json" with the pusher field from appsettings.json in discord for it to work

### CODE REVIEW

**Comment by Jaffenheimer** [comment]
Api-keys have been revoked so they are no longer valid, and the "Pusher field" has been removed from the appsettings.json file

**Comment by NikolajKrogh** [comment]
Closes #44

File: backend/p8mobility.restapi/Controllers/AdminController.cs [hunk 1]
```diff
@@ -145,5 +146,15 @@ public async Task<IActionResult> UpdatePeopleAmount(int amount, Guid busStopId)
         Program._stateController.UpdatePeopleCount(busStopId, amount);
         return Ok($"Successfully updated people amount on bus stop with id: {busStopId} to {amount}");
     }
-    
+
+    [HttpPost("Pusher/Test")]
```
Comment (JJTM20) [inline backend/p8mobility.restapi/Controllers/AdminController.cs:150]: Går ud fra test bliver ændret?

**Review by JJTM20 — APPROVED** [review]

**Comment by JJTM20** [comment]
lgtmq+

**Pull request merged**